features/step_definitions/import.ts

```diff
@@ -8,6 +9,13 @@ Given(
   },
 );
 
+Given(
+  "There is a file {string} with content: {string}",
```

Comment:
Following the official document, I think we can use the below step definition to make it more clear
https://github.com/cucumber/cucumber-js/blob/main/docs/support_files/step_definitions.md#cucumber-expressions

```suggestion
  "I have a file {string} with content: {string}",
```